File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.cs
public class Solution {
    public int TriangularSum(int[] nums) {
        int k=0;
        while(k<nums.Length){
            for(int i=1;i<nums.Length-k;i++){
                nums[i-1]=(nums[i]+nums[i-1])%10;
            }
            k++;
        }
        return nums[0];
    }
}

File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function triangularSum($nums) {
        $k=0;
        while($k<count($nums)){
            for($i=1;$i<count($nums)-$k;$i++){
                $nums[$i-1]=($nums[$i]+$nums[$i-1])%10;
            }
            $k++;
        }
        return $nums[0];
    }
}